phfs.h

```diff
@@ -19,29 +21,54 @@
 #include "config.h"
 
 typedef struct {
-	s32 h;
-	void *offs;
-} handle_t;
+	unsigned int pn; /* phfs device number descriptor */
+	unsigned int fd; /* file descriptor               */
```

Comment:
It think we agreeg on "file handle" nomenclature

Reply:
I remember that we should use "device handle" instead of "device descriptor" so probably I should change comment `/* phfs device number descriptor */` to `/* phfs device handle */`.
However, we didn't have discussion about files. I assumed that **file descriptor** is a common phrase and it can be applied in this case.

Reply:
We should change it to the handle. Semantically file descriptor plays role of resource handle which points to the operating system resource shared among running processes. In this case we have no processes but to be inline with our agreements and  be consistent I suggest not to introduce new terms.